<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="UserrlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContralineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RepetirContlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>711</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imganes.qrc">:/FONDOLO.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>UserrlineEdit</zorder>
  <zorder>ContralineEdit</zorder>
  <zorder>RepetirContlineEdit</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources>
  <include location="imganes.qrc"/>
 </resources>
 <connections/>
</ui>
